static/docs/commands-reference/remote_add.md

```diff
@@ -18,18 +18,18 @@ usage: dvc remote add [-h] [--global] [--system] [--local] [-q | -v]
 
 positional arguments:
   name           Name of the remote.
-  url            URL. (See supported URLs below.)
+  url            URL. (See supported URLs in the examples below.)
 ```
 
 ## Description
 
 `name` and `url` are required. `url` specifies a location to store your data. It
 could be S3 path, SSH path, Azure, Google cloud, Aliyun OSS local directory,
 etc. (See more examples below.) If `url` is a local relative path, it will be
-resolved relative to the current directory but saved **relative to the config
-file location** (see LOCAL example below). Whenever possible DVC will create a
-remote directory if it doesn't exists yet. It won't create an S3 bucket though
-and will rely on default access settings.
+resolved relative to the present working directory but saved **relative to the
```

Comment:
double check that it's actually "current directory", i.e. some directory inside the workspace

Reply:
I changed all the occurrences of the term "current directory" with "present working directory" as I think they are synonyms (in order to standardize, using typical Linux jargon). Are they not synonyms?

Reply:
As for the description yes that's how it works, I just double checked.